lldb/source/Plugins/ObjectFile/PECOFF/WindowsMiniDump.cpp

```diff
@@ -21,11 +21,11 @@
 namespace lldb_private {
 
 bool SaveMiniDump(const lldb::ProcessSP &process_sp,
-                  const lldb_private::FileSpec &outfile,
-                  lldb_private::Status &error) {
+                  const CoreDumpOptions &core_options, lldb_private::Status &error) {
   if (!process_sp)
     return false;
 #ifdef _WIN32
+  const auto outfile = core_options.GetOutputFile();
```

Comment:
This is a std::optional<FileSpec> and will not work below. You must `unrwap` it:
```
// The FileSpec is already checked in PluginManager::SaveCore.
assert(core_options.GetOutputFile().has_value()); 
const FileSpec &outfile = core_options.GetOutputFile().value();
```